### ISSUE TITLE
auth credentials error trying to build docker file

### ISSUE THREAD

**Issue by urlbox-io** [nisaacson/aws-lambda-xvfb#1]
Hey there, I tried following the instructions, setup my own ecr registry and logged in, then tried to build the docker file but get:

    ➜  aws-lambda-xvfb git:(master) ✗ docker build -t aws-lambda-xvfb .
    Sending build context to Docker daemon  113.9MB
    Step 1/16 : FROM 137112412989.dkr.ecr.us-west-2.amazonaws.com/amazonlinux:latest
    Get https://137112412989.dkr.ecr.us-west-2.amazonaws.com/v2/amazonlinux/manifests/latest: 
    no basic auth credentials

I'm guessing because the dockerfile is trying to pull from a private repo base image?

**Comment by kylegato** [comment]
@urlbox-io yeah I ran into the same thing. You can try removing the "137112412989.dkr.ecr.us-west-2.amazonaws.com/" section and it'll pull from https://hub.docker.com/_/amazonlinux/
